core/src/main/resources/lib/hudson/propertyTable.jelly

```diff
@@ -63,14 +66,20 @@ THE SOFTWARE.
           <td style="white-space: normal" class="property-table__value">
             <j:choose>
               <j:when test="${attrs.sensitive}">
-                <div class="hidden-info">
-                  <div class="jenkins-button">
```

Comment:
`button` should be used or at least a link, I tested in voiceover and there was 0 feedback given to the user that this was clickable as a div (I've changed it to a button)